### PR TITLE
Fixed typo, grammar and order of contents

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,24 +15,26 @@
...
 
 **Investigate Command & Control:**
 
-Launch "Investigate Command & Control" playbook and observe various investigation activities such as
+Launch **Investigate Command & Control** playbook and observe various investigation activities such as:
 
-- Enriches alerts for command-and-control behaviour by identifying the reputation of related artefacts such as source IP addresses and file hashes.
-- Find Geolocation of Source IP.
-- Investigates any anomalous processes running on the host on which the attack has occurred are terminated using the CarbonBlack Response
+- Enriched alerts for command-and-control behavior by identifying the reputation of related artifacts such as source IP addresses and file hashes.
```
Comment (cs-suyog-jinturkar) [inline README.md]: Enriches is correct here.

File: README.md [hunk 2]
```diff
@@ -15,24 +15,26 @@
...
+- Geolocation of Source IP.
+- Anomalous processes running on the host, which was the target of the attack, are terminated using the CarbonBlack Response
```
Comment (cs-suyog-jinturkar) [inline README.md]: Investigates any anomalous --> is also correct

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added review comments

File: README.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (cs-abhishek-shukla) [reply README.md]: Fixed.

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**